projects/batfish/test/org/batfish/main/BatfishTest.java

```diff
@@ -0,0 +1,91 @@
+package org.batfish.main;
+
+import org.batfish.common.BatfishException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Test for {@link Batfish#listAllFiles(Path)}.
```

Comment:
This is too specific: the test class will eventually test more functions. "Tests (since plural) for {@link Batfish}." is right.